sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py

```diff
@@ -35,6 +35,9 @@ def dut_setup(dut, settings, testdata, request) -> Bond:
 
     mode = request.param["mode"]
     explicit_mac = request.param["mac"]
+    if mode == 1 and explicit_mac:
+        pytest.xfail("Expected failure - mode 1 with explict bond mac. \
```

Comment:
The indentation make it look like two line of code. How about,
pytest.xfail("Expected failure - mode 1 with explicit bond mac."
                     "That should work."
                    )

Also, "explict" should be spelled as "explicit"

Reply:
Somehow I can't format above text right, but you get the point